app/templates/index.hbs

```diff
@@ -18,9 +18,10 @@
           {{#each this.model as | repo |}}
             <EsLinkCard
               class="lg:col-3 bg-dark"
-              @href={{repo.htmlUrl}}
+              @href="{{repo.htmlUrl}}/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22"
               @title={{repo.displayName}}>
               <p>Forks: {{#if (not repo.forks)}} No forks {{else}} {{repo.forks}} {{/if}}</p>
+              <p>All open issues: {{repo.openIssues}}</p>
```

Comment:
I noticed that the number of open issues is incorrectly defined. For example, the deploy preview showed 92 open issues for `guides-source`, but I saw 88 issues instead.

What seems to be happening is `repo.openIssues` is actually equal to the number of open issues plus that of pull requests. Until we can figure out how to fix this issue, I don't think it's a good idea to show the number yet.

```suggestion
```